sources/packages/web/src/views/student/AppStudent.vue

```diff
@@ -18,25 +18,43 @@ import { StudentService } from "../../services/StudentService";
   },
 })
 export default class AppStudent extends Vue {
```

Comment:
Consider refactor this component to be a "function component" instead of a class component to be aligned with the others along with this application. I am suggesting this in this PR because it is receiving more logic and it seems to be a good time to do it.

Reply:
DONE

Reply:
I am still seeing it as a class component "export default class AppStudent extends Vue".
Am I missing something?

Reply:
Sorry, I missed this part.